week03/html/contact.html

```diff
@@ -0,0 +1,51 @@
+<link href="../css/contact.css" rel="stylesheet" /> 
+<link rel="icon" type="image/png" sizes="16x16" href="../src/billyLogo.png"/>
+
+<title>Lee Gabin</title>
+
+<body>
+    <!--상단바-->
+
+    <header id="header">
+        <div class="navbar">
+        <div class="nav1">
+        <a href="../html/home.html" class="logo">
+        <img src="../src/mainLogo.png" height="70px" />
+    </a>
+    </div>
+    <div class="nav2"></div>
+    <div class="nav3">
+    <ul>
+        <li><a href="../html/home.html">Home</a></li>
+        <li><a href="../html/about.html">About</a></li>
+        <li><a href="../html/project.html">Project</a></li>
+        <li><a href="../html/activity.html">Activity</a></li>
+        <li><a href="../html/contact.html" style="color: black;">Contact</a></li>
+    </ul>
+    </div>
+    </div>
+    </header>
+
+    <!--내용-->
+    <div class="contactMe">
+        <div class="title">Contact</div>
+        <div class="mail">
+            <a class="t1">E-MAIL</a><br>

```

Comment:
1. 텍스트 스타일을 적용하기 위한 용도로는 `a` 태그보다 `span` 태그가 적합합니다.
2. `br` 태그로 줄바꿈을 하는 것보다는 css에서 한 번에 디자인을 관리하는 것이 좋습니다. css에서는 다음과 같이 요소 줄바꿈을 할 수 있습니다.
```css
body main ul li {
    display: flex;
    flex-flow: column;
}
```

```suggestion
            <span>E-MAIL</span>
```